src/server/local/index.js

```diff
@@ -169,16 +169,16 @@ export default server => {
       sendDataFile(res, `${dataFilePath}.json`, next);
     })
     .get(topicDataPath, async ({ params }, res, next) => {
-      const { service, variant, id } = params;
+      const { service, variant = '', id } = params;
```

Comment:
would there be benefit of readability to put variant at the end now that it has a default setting? Not sure what our convention is, tbh..

Reply:
Makes sense, it's good practice to do this